plugins/statistics.koplugin/main.lua

```diff
@@ -14,6 +14,7 @@ local ReaderProgress = require("readerprogress")
 local ReadHistory = require("readhistory")
 local Screensaver = require("ui/screensaver")
 local SQ3 = require("lua-ljsqlite3/init")
+local SyncService = require("frontend/apps/cloudstorage/syncservice")
```

Comment:
May this module load some other cloudstorage/ modules?
If yes, you may want to delay its require/loading until the first time it is needed (https://github.com/koreader/koreader/pull/9532#issuecomment-1251309379).

Reply:
No, that is loaded in a callback function. The module requires mostly harmless other parts, I believe.
https://github.com/koreader/koreader/blob/0ee2dd92353a17892a87f3ea6db5b238be684784/frontend/apps/cloudstorage/syncservice.lua#L1-L12